WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -473,6 +473,18 @@ function WeakAuras.regionPrototype.AddExpandFunction(data, region, cloneId, pare
     WeakAuras.Animate("display", data, "main", data.animation.main, region, false, nil, true, cloneId);
   end
 
+  function region:OptionsClosed()
```

Comment:
Our leeches who want an Options closed event will be happy about this, I think 😅

Reply:
:D